src/cli/manager.rs

```diff
@@ -49,6 +50,33 @@ pub fn server_address() -> &'static str {
         .unwrap();
 }
 
+// Return the desired www path
+pub fn www_path() -> &'static str {
+    return MANAGER
+        .as_ref()
+        .clap_matches
+        .value_of("www-path")
+        .unwrap_or_else(|| {
+            let www_paths = vec![
+                "./www",
```

Comment:
This will the path that's being used to call the binary and not the path of the binary itself.
E.g: If I call from `/` the binary `/home/patrick/project/mavlink-camera-manager`, the program will search for these paths under `/` and not under `/home/patrick/project/www`

To fix that, you should use [std::env::current_exe](https://doc.rust-lang.org/std/env/fn.current_exe.html#), get the folder path and add `www` to the final result.